DQM/Integration/python/clients/pixel_dqm_sourceclient-live_cfg.py

```diff
@@ -102,9 +107,34 @@
 if (process.runType.getRunType() == process.runType.hi_run):    
     process.load('Configuration.StandardSequences.ReconstructionHeavyIons_cff')
     process.load('Configuration.StandardSequences.RawToDigi_Repacked_cff')
-    process.siPixelDigis.InputLabel   = cms.InputTag("rawDataRepacker")
-
+    process.castorDigis.InputLabel = cms.InputTag("rawDataRepacker")
+    process.csctfDigis.producer = cms.InputTag("rawDataRepacker")
+    process.dttfDigis.DTTF_FED_Source = cms.InputTag("rawDataRepacker")
+    process.ecalDigis.InputLabel = cms.InputTag("rawDataRepacker")
+    process.ecalPreshowerDigis.sourceTag = cms.InputTag("rawDataRepacker")
+    process.gctDigis.inputLabel = cms.InputTag("rawDataRepacker")
+    process.gtDigis.DaqGtInputTag = cms.InputTag("rawDataRepacker")
+    process.gtEvmDigis.EvmGtInputTag = cms.InputTag("rawDataRepacker")
+    process.hcalDigis.InputLabel = cms.InputTag("rawDataRepacker")
+    process.muonCSCDigis.InputObjects = cms.InputTag("rawDataRepacker")
+    process.muonDTDigis.inputLabel = cms.InputTag("rawDataRepacker")
+    process.muonRPCDigis.InputLabel = cms.InputTag("rawDataRepacker")
+    process.scalersRawToDigi.scalersInputTag = cms.InputTag("rawDataRepacker")
+    process.siPixelDigis.InputLabel = cms.InputTag("rawDataRepacker")
+    process.siStripDigis.ProductLabel = cms.InputTag("rawDataRepacker")
+
+    #replace the effect of era in local reco
+    process.siPixelDigis.UsePhase1=True
+    process.siPixelClusters.VCaltoElectronGain      = cms.int32(47)   # L2-4: 47 +- 4.7 
+    process.siPixelClusters.VCaltoElectronGain_L1   = cms.int32(50)   # L1:   49.6 +- 2.6 
+    process.siPixelClusters.VCaltoElectronOffset    = cms.int32(-60)  # L2-4: -60 +- 130 
+    process.siPixelClusters.VCaltoElectronOffset_L1 = cms.int32(-670) # L1:   -670 +- 220 
+    process.siPixelClusters.ChannelThreshold        = cms.int32(10) 
+    process.siPixelClusters.SeedThreshold           = cms.int32(1000) 
+    process.siPixelClusters.ClusterThreshold        = cms.int32(4000) 
+    process.siPixelClusters.ClusterThreshold_L1     = cms.int32(2000) 
```

Comment:
These should probably be `siPixelClustersPreSplitting`.

Reply:
here is the printout with python <pixel client> runkey=hi_run

Source: cms.Source("DQMStreamerReader",
    SelectEvents = cms.untracked.vstring('*'),
    delayMillis = cms.untracked.uint32(500),
    deleteDatFiles = cms.untracked.bool(False),
    endOfRunKills = cms.untracked.bool(True),
    minEventsPerLumi = cms.untracked.int32(1),
    nextLumiTimeoutMillis = cms.untracked.int32(240000),
    runInputDir = cms.untracked.string('/tmp'),
    runNumber = cms.untracked.uint32(111),
    scanOnce = cms.untracked.bool(False),
    skipFirstLumis = cms.untracked.bool(False),
    streamLabel = cms.untracked.string('streamDQM')
)

Loaded configuration file from: []
dqmRunConfig: cms.PSet(
    collectorHost = cms.untracked.string('dqm-c2d07-29.cms'),
    collectorPort = cms.untracked.int32(9190),
    type = cms.untracked.string('userarea')
)
Traceback (most recent call last):
  File "pixel_dqm_sourceclient-live_cfg.py", line 92, in <module>
    process.load("Configuration.StandardSequences.RawToDigi_Repacked_cff")
  File "/opt/offline/slc7_amd64_gcc630/cms/cmssw/CMSSW_9_2_12/python/FWCore/ParameterSet/Config.py", line 582, in load
    module = __import__(moduleName)
  File "/opt/offline/slc7_amd64_gcc630/cms/cmssw/CMSSW_9_2_12/python/Configuration/StandardSequences/RawToDigi_Repacked_cff.py", line 10, in <module>
    gtEvmDigis.EvmGtInputTag = 'rawDataRepacker'
NameError: name 'gtEvmDigis' is not defined